### ISSUE TITLE
Set smooth curves per edge?

### ISSUE THREAD

**Issue by Lukather1971** [almende/vis#335]
Hi
In my configuration i setup smoothCurves:false because i dont want put all edges with "quadratic Bezier curves" smoothcurves , instead i like to have straight lines between 2 nodes 
But sometimes i have 2 directions between 2 nodes. When i do this i can not select the first edge direction but only the last edge. In addition labels overlapping each other.

for example U can reproduce with this and try to get Edge A with Label A again!:
Configuration:
smoothcurves:false
Node A
Node B
1 Edge: from A to B with label "Label  A" style: arrow or arrow-center
2 Edge form B to A with label "Label B" style: arrow-center

So i was looking in the documentation i try to find a sollution like smootCurves only on particular edges in the "Edges configuration" section but not have found it.
Is this A Bug or enhancement for next release?
See my example below:
ps: for the record i use version 3.4.2
![2 direction](https://cloud.githubusercontent.com/assets/8628365/4347778/e24d3ece-4169-11e4-8b4b-b9e9f3270730.png)

**Comment by AlexDM0** [comment]
Hi,

Over the last year a lot of feature requests have been made. We have just introduced our new website which has a list of the requested features. We have placed this request on that list. 

The list can be found here:
http://visjs.org/featureRequests.html

An explaination of the new system can be found here:
http://visjs.org/blog.html#New\ website\ for\ vis\.js\!

I would like to stress that this does not mean we abandon this request. Discussion here will continue if needed on this feature but we will close it to keep our Github issue page more of a bug-todo list.

Future feature requests will still be made here and then added to the website by us.

Regards,

Alex
